src/protocols/chainsync.rs

```diff
@@ -194,6 +194,8 @@ impl Protocol for ChainSyncProtocol {
                     chain_blocks.push((4492799, hex::decode("f8084c61b6a238acec985b59310b6ecec49c0ab8352249afd7268da5cff2a457").unwrap()));
                     // Last byron block of testnet
                     chain_blocks.push((1598399, hex::decode("7e16781b40ebf8b6da18f7b5e8ade855d6738095ef2f1c58c77e88b6e45997a4").unwrap()));
+                    // Last byron block of guild
```

Comment:
@pavlix , The next refactor you do, we should probably move this stuff outside the library. I should be injecting this in from the CNCLI side through a callback or something.